README.md

```diff
@@ -54,6 +54,21 @@ To increase logging level to include info and debug messages, set the environmen
 `DMCI_LOGLEVEL` to the desired level. Valid levels are `CRITICAL`, `ERROR`, `WARNING`, `INFO`, and
 `DEBUG`.
 
+## Usage
+
+First initialize the config.yaml or use environment-variables. Then, to start the API run:
+
+```python
+python dmci_start_api.py
+```
+
+Then you can post with curl to the api - endpoint:
+
+```bash
+curl --data "@<PATH_TO_MMD_FILE>" localhost:5000/v1/insert
+```
+insert is currently the only implemented command. Other commands will return 
```

Comment:
Return? What? :)

What about the different return codes 200/400 and so on?

Reply:
Oh, thanks I'll fix.

Reply:
Was there a status page i could look at too? Healtz or something similar?